File: java-selenide-gradle-allure/src/test/java/TestClass2.java
import io.qameta.allure.Description;
import org.junit.jupiter.api.Test;

import static com.codeborne.selenide.Condition.text;
import static com.codeborne.selenide.Selectors.by;
import static com.codeborne.selenide.Selenide.$;

public class TestClass2 extends BaseTest {
    @Test
    @Description("Failure example")
    void failedTest() {
        $(by("name", "q")).setValue("allure-nunit").pressEnter();
        $(".repo-list li h3").shouldHave(text("unickq"));
    }
}

File: kotlin-selenide-gradle-allure/src/test/kotlin/TestClass2.kt
import com.codeborne.selenide.Condition
import com.codeborne.selenide.Selenide
import io.qameta.allure.Feature
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.TestInstance
import org.openqa.selenium.By

@TestInstance(TestInstance.Lifecycle.PER_CLASS)
class TestClass2 : BaseTest() {

    @Test
    @Feature("Search")
    fun testFail() {
        Selenide.`$`(By.name("q")).setValue("allure-nunit").pressEnter()
        Selenide.`$`(".repo-list li h3")
            .shouldHave(Condition.text("unickq"))
    }
}